src/blueprint/into_schema.rs

```diff
@@ -102,11 +160,23 @@ fn to_type(def: &Definition) -> dynamic::Type {
                       const_value
                     }
                   }
-                  _ => expr.eval(&ctx, &Concurrent::Sequential).await?,
+                  _ => {
+                    if let Ok(const_value) = read_entity_cache(&ctx, &of_type).await {
+                      read_from_entity_cache = true;
+                      log::info!("Reading to cache.");
+                      const_value
+                    } else {
+                      expr.eval(&ctx, &Concurrent::Sequential).await?
+                    }
+                  }
                 };
 
+                if !read_from_entity_cache {
+                  write_entity_cache(&ctx, &of_type, &const_value);
+                }
```

Comment:
sounds like we gonna to write to cache without checking that cache is enabled. Is it ok?

Reply:
the function `write_entity_cache` is doing that checking